scancodeio/urls.py

```diff
@@ -52,6 +52,7 @@
     path("admin/", admin.site.urls),
     path("api/", include(api_router.urls)),
     path("license/", include(licenses.urls)),
+    path("scan/", include("scantext.urls")),
```

Comment:
`scan` is a bit to generic, let's use `scantext` in the url as well.

Reply:
Sure, will change it to `scantext` in the upcoming commit